### PR TITLE
Release v0.5.1

### PR DESCRIPTION
**Pull request by roidelapluie**
@SuperQ @beorn7 

Now that #32 is merged, the plan is to release a bugfix release of Prometheus with this.

I'd like to cut a v0.5.1 release of the toolkit in such a way that I can resume my work on #24, which would end up in a next release of the toolkit but not in the prometheus 2.24 release.

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.5.1 / 2021-01-15
+
+* [IMPROVEMENT] Cache basic authentication results to significantly improve
```
Comment (beorn7) [inline CHANGELOG.md:3]: Don't we usually call this `[ENHANCEMENT]`?

Comment (roidelapluie) [reply CHANGELOG.md]: Yes we do :)

Comment (roidelapluie) [reply CHANGELOG.md]: Also should I write `performances` or `performance`?

Comment (beorn7) [reply CHANGELOG.md]: I'd say `performance` is better.

`performances` sounds like a number of dance performances or something. :-)

Comment (roidelapluie) [reply CHANGELOG.md]: 👯

**Review by beorn7 — APPROVED** [review]

**Comment by roidelapluie** [comment]
We still need #39 so we do not introduce a side-channel timing vulnerability.

**Review by SuperQ — APPROVED** [review]

**Pull request merged**